### PR TITLE
Cyan

### PR DESCRIPTION
**Pull request by anibor164**
Oh, hey look a pull request!

### CODE REVIEW

File: default.htm [hunk 1]
```diff
@@ -1,255 +1,527 @@
...
-						e.preventDefault();
-						e.stopPropagation();
-					}
-					if ("12345".includes(e.key)) {
-						document.getElementById("letters" + e.key).focus();
-					}
+                document.getElementById("rWires").innerHTML = "Total wires: " + totalcount + ". Cut the " + result + " wire."; 
+			}
+
+			function jump(e) { //This is specific to the password solver? Should be fine for now but if there are additional text inputs this may need context.
```
Comment (halojenproductions) [inline default.htm:503]: Yup I haven't got around to making their names more elaborate now that there are other kinds of solvers.

File: default.htm [hunk 2]
```diff
@@ -1,255 +1,527 @@
...
+		#button_colour4:not(:checked)+.btn {
+			color: #dc3545;
+		}
+
+		#button_colour4:checked+.btn {
+			background-color: #dc3545;
+		}
+
+		.btn-group-sm {
+			margin-top: 6px; /*TODO : better solution to spacing*/
```
Comment (halojenproductions) [inline default.htm:68]: There's a bootstrap way of doing this.

Comment (anibor164) [reply default.htm]: Figured, I just forgot to check my TODOs

File: default.htm [hunk 3]
```diff
@@ -127,129 +136,395 @@ <h3>Keep Talking and Nobody Explodes</h3>
...
-						let wordLetters = lookupWord.split('');
-
-						let hasMatch = true;
-						for (var i = 0; i < 5; i++) {
-							hasMatch &&= inputs[i].length === 0 || inputs[i].includes(wordLetters[i]);
-						}
-						return hasMatch;
+					let hasMatch = true;
+					for (var i = 0; i < 5; i++) { //I've been drinking but could you replace '5' condition with inputs[i].length and remove the === 0 below? SM
+						hasMatch &&= inputs[i].length === 0 || inputs[i].includes(wordLetters[i]);
```
Comment (halojenproductions) [inline default.htm:396]: You're getting muddled up between inputs[i].length and inputs.length.
This like could be changed to `for (var i = 0; i < inputs.length; i++) {` but the one below is getting the number of chars in inputs[i] to see if any have been put in.

Comment (anibor164) [reply default.htm:396]: Yeah, I read this today and I was wrong.

**Review by halojenproductions — APPROVED** [review]

**Pull request merged**